roles/kalite/tasks/install.yml

```diff
@@ -20,9 +20,15 @@
   when: not (is_debian_12 or is_ubuntu_2304 or is_ubuntu_2310)
   # 2020-03-31: Testing for {is_raspbian_9, is_raspbian_10} is not currently nec, as testing for {is_debian_9, is_debian_10} covers that already.
 
+# libpython2.7-stdlib from ubuntu-22.04 used in install_python2.sh amd64 is compiled against libssl3 and libffi8
+- name: Force install libssl1.1 for python2 on non x86_64 machines
+  include_role:
+    name: libssl1
+  when: is_raspbian_12 and ansible_architecture == "armhf" and libssl1_installed is undefined
```

Comment:
Question: does KA Lite need libssl1.1 on every arch — i.e.every OS where libssl1.1 is absent?

If that's true (Y/N ?) then this would cover RasPiOS 12 and pure Debian 12 and Mint 21 and Ubuntu 22.04+:

```suggestion
  when: libssl1_installed is undefined and (is_debian_12 or is_linuxmint_21 or is_ubuntu and os_ver is version('ubuntu-2204', '>='))
```

Reply:
> Question: does KA Lite need libssl1.1 on every arch — i.e.every OS where libssl1.1 is absent?

Not at all when using the Ubuntu repos, see the note 'python2 on ubuntu-22.04 is compiled against libssl3 and libffi8', that allows the use of the repos as is without fiddling with libssl1.1 in the later commits

 
> If that's true (Y/N ?) then this would cover RasPiOS 12 and pure Debian 12 and Mint 21 and Ubuntu 22.04+:

The arm64 and amd64 arches would be covered by using the U-22.04 repos. Might have to adjust for Mint 21 I'll look in a bit

Reply:
Mint 21 would be using the '(for Python 2) on older OSs' by virtue of the conditional 
when: not (is_debian_12 or is_ubuntu_2304 or is_ubuntu_2310)

Mint-21 is still based on U-22.04 correct?

Reply:
> Mint-21 is still based on U-22.04 correct?

Yes.  I'm assuming Mint 21 is permanently based on (evolving) versions of Ubuntu 22.04.x

But that might possibly be slightly off.

Reply:
Doesn't really matter as long python2 is available from their repos, the old code path would still work.